nursery/write-to-console.yml

```diff
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: write to console
```

Comment:
Merge with the console rule above? The APIs seem to line up.

Reply:
Good point; I'll make this change pending response to my comments on the referenced `manipulate console` rule.